### PR TITLE
Drop yast2-packager-devel-doc package

### PR DESCRIPTION
**Pull request by imobach**
Drop yast2-packager-devel-doc package ([fate#320356](https://fate.suse.com/320356)).

### CODE REVIEW

File: package/yast2-packager.spec [hunk 1]
```diff
@@ -87,18 +87,9 @@ Requires:       yast2-ruby-bindings >= 1.0.0
 Summary:	YaST2 - Package Library
```
Comment (jreidinger) [inline package/yast2-packager.spec]: I do not see here version increase

Comment (imobach) [inline package/yast2-packager.spec]: You're right. Now it's done.

Comment (jreidinger) [inline package/yast2-packager.spec]: still do not see it

Comment (imobach) [inline package/yast2-packager.spec]: It's [here](https://github.com/yast/yast-packager/pull/161/commits/3b40ea7e6ee76a67d8cf5856ad6cc097612a032d).

**Comment by jreidinger** [comment]
LGTM

**Comment by imobach** [comment]
Thanks!

**Pull request merged**